CMakeLists.txt

```diff
@@ -506,6 +506,9 @@ add_subdirectory(experimental/experimental_examples)
 
 # Build tools
 if (TILEDB_TOOLS)
+  if (TILEDB_AZURE AND NOT TILEDB_VCPKG)
+    message(FATAL_ERROR "Building tools with Azure enabled requires enabling TILEDB_VCPKG")
```

Comment:
I had linker errors. I suspect its because of the use of the interface objects, and a ordering issue with the order of the static libs being included. Since `tools` are not used in conda (nor really anywhere) I decided to make it require vcpkg.